components/combobox/__examples__/snapshot/base-open-class-name.jsx

```diff
@@ -45,6 +45,7 @@ class Example extends React.Component {
 		return (
 			<IconSettings iconPath="/assets/icons">
 				<Combobox
+					aria-describedby="described-by-unique-id"
```

Comment:
What is the reason to add `aria-describedby="described-by-unique-id"` to all snapshots instead of just the "required input" example?

Reply:
If I don't specify it on all combobox snapshot tests, it will generate a unique ID. That will fail the snapshot tests, right? Or should I get rid of the OR statement in 
```
getErrorId () {
  return `${this.props['aria-describedby'] ||
    this.generatedId}-error-message`;
}
```

Reply:
See Input's https://github.com/salesforce/design-system-react/blob/master/components/forms/input/index.jsx#L266